Extension/lib/content-script/assistant/js/adguard-rules-constructor.js

```diff
@@ -95,17 +95,32 @@ var AdguardRulesConstructorLib = (function (api) {
     };
 
     var getElementClassName = function (element) {
-        var className = '';
+        var wrap = false;
+
+        var classes = [];
         var classList = element.classList;
         if (classList && classList.length > 0) {
             for (var i =0; i < classList.length; i++) {
                 var c = classList.item(i);
                 if (c) {
-                    className += '.' + c;
+                    classes.push(c);
+
+                    if (c.indexOf('.') > 0) {
+                        wrap = true;
+                    }
                 }
             }
         }
 
+        var className = '';
+        if (classes.length > 0) {
+            if (wrap) {
+                className = '[class="' + classes.join(' ') + '"]';
```

Comment:
You lost spaces in 'class' attribute value. Values in 'classList' property doesn't contains spaces.